nautobot/extras/tests/example_jobs/test_file_upload_fail.py

```diff
@@ -21,7 +26,7 @@ class Meta:
 
     def run(self, file):
         contents = str(file.read())
-        self.log_warning(message=f"File contents: {contents}")
+        logger.warning("File contents: %s", contents)
```

Comment:
Should any/all of these test jobs use `self.logger` instead of `logger`?

Reply:
Both of them are supported since they both return the same logger. We need to set `self.logger` to use on some of our internal methods in `BaseJob` so it needs to stay but I think we should document the usage of `get_task_logger` since it's closer to the stdlib logging api.